app/Http/Transformers/SignupTransformer.php

```diff
@@ -26,6 +26,7 @@ class SignupTransformer extends TransformerAbstract
      */
     public function transform(Signup $signup)
     {
+        dd($signup->events);
```

Comment:
That being said I'm a little confused by how all these relationships are set up. It seems like a kinda tangled web. 🕸 It might be nice to chat it out IRL at some point.